setup.py

```diff
@@ -23,8 +23,10 @@
     packages=["arrow"],
     zip_safe=False,
     python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
-    install_requires=["python-dateutil"],
-    extras_require={":python_version=='2.7'": ["backports.functools_lru_cache>=1.2.1"]},
+    install_requires=[
+        "python-dateutil",
+        "backports.functools_lru_cache;python_version=='2.7'",
```

Comment:
Is it ok that we've removed the >=1.2.1 requirement here? Now it could install an older version.

Reply:
I locked it to >=1.2.1 just in case :)